docs/reference/release-notes/8.17.2.asciidoc

```diff
@@ -48,4 +48,5 @@ Snapshot/Restore::
 Ingest Node::
 * Improve memory aspects of enrich cache {es-pull}120256[#120256] (issues: {es-issue}96050[#96050], {es-issue}120021[#120021])
 
-
+* `VALUES` aggregate function can run for a long, long time when collecting many, many groups. Hundreds of thousands of groups can run on one thread for many minutes and millions of groups run for days. It is O(n^2^) with the number of groups. These will not respond to the tasks cancellation API the whole time. Fixed by {es-pull}123073[#123073] and available in 8.16.5, 8.17.3, 8.18.0, and all releases after that.
```

Comment:
This is already in the release notes for 8.17, but the 8.x branch doesn't have it, yet.